source/federation.rst

```diff
@@ -0,0 +1,410 @@
+HEPiX image lists format.
+
+Usage of resources is gathered centrally using EGI Accounting repository and
+available for visualisation at  `EGI Accounting portal <https://accounting.egi.eu>`_.
+
+Those endpoints published in the EGI Configuration Database are monitored via
+`ARGO <https://argoeu.github.io/>`_. The set of probes check the availability
+of the providers and their correct functionality. 
+
+Users and Community platforms built on top of the EGI IaaS can interact with
+the cloud providers at three different layers:
+
+* Directly using the IaaS APIs to manage individual resources. This option is
+  recommended for pre-existing use cases with requirements on specific APIs.
+
+* Using IaaS Federated Access Tools that allow managing the complexity of
+  dealing with different providers in a uniform way. These tools include:
+
+  * IaaS provisioning systems that allow to define infrastructure as code and 
+    manage and combine resources from different providers, thus enabling the
+    portability of application deployments between them (e.g. IM or Terraform); 
+    and
+
+  * cloud brokers, that provide matchmaking for workloads to available
+    providers (e.g. the INDIGO-DataCloud Orchestrator).
+
+* Using the AppDB VMOps dashboard, a web-based GUI that simplifies the
+  management of VMs on any provider of the EGI infrastructure. AppDB VMOps in
+  turn relies on the Infrastructure Manager.
+
+EGI provides ready-to-use software components to enable the federation for
+OpenStack and OpenNebula. These components rely on public APIs of the
+IaaS system and use Check-in accounts for authenticating into the provider.
+
+
+Implementation
+--------------
+
+AAI
+^^^
+
+Federated identity ensures that users of the federation can use a single
+account for accessing the resources.
+
+OpenID Connect
+::::::::::::::
+
+Providers of the EGI Cloud support authentication with OAuth2.0 tokens
+provided by Check-in OpenID Connect Identity provider. Support builds on
+the `AAI guide for SPs <https://wiki.egi.eu/wiki/AAI_guide_for_SPs>`_ with
+detailed configuration provided at the `EGI Cloud integration manual <https://egi-federated-cloud-integration.readthedocs.io/en/latest/openstack.html#openid-connect-support>`_.
+
+The integration relies on the OpenStack Keystone `OS-FEDERATION API <https://developer.openstack.org/api-ref/identity/v3-ext/index.html#os-federation-api>`_.
+
+Legacy VOMS / X.509 certificates
+::::::::::::::::::::::::::::::::
+
+EGI can support those users still using X.509 certificates extended
+with VO attributes (e.g. acknowledging that the user is member of the VO)
+in a so called VOMS proxy. This VOMS proxy certificate is used in subsequent
+calls to the endpoints which map the certificate and VO information
+via specific integration modules for VOMS authentication.
+
+There are two implementations for the support of VOMS proxies:
+
+* `OpenNebula Perun integration <https://github.com/EGI-FCTF/fctf-perun>`_ 
```

Comment:
This shouldn't be in EGI-FCTGF, we shouldn't refer to an org described as deprecated.

Reply:
Linked to Keystorm instead